CMakeLists.txt

```diff
@@ -20,30 +13,27 @@ add_library(gfx STATIC gfx.cpp)
 set(GFX_DXC_PATH ${CMAKE_CURRENT_SOURCE_DIR}/third_party/dxc)
 set(GFX_PIX_PATH ${CMAKE_CURRENT_SOURCE_DIR}/third_party/WinPixEventRuntime)
 
-set(GFX_DXC_PATH ${GFX_DXC_PATH} PARENT_SCOPE)
-set(GFX_PIX_PATH ${GFX_PIX_PATH} PARENT_SCOPE)
+if(NOT PROJECT_IS_TOP_LEVEL)
+    set(GFX_DXC_PATH ${GFX_DXC_PATH} PARENT_SCOPE)
+    set(GFX_PIX_PATH ${GFX_PIX_PATH} PARENT_SCOPE)
+endif()
```

Comment:
Takahiro remove this in his PR but instead I just put them in a conditional. If you run cmake on gfx alone (i.e. its the top level project) then these lines used to throw warnings about non-existent parent scope. Checking for a higher level fixes this